configure.ac

```diff
@@ -443,6 +443,10 @@ AM_CONDITIONAL([HAVE_LIBFTD2XX], [test "x$have_libftd2xx" = xyes])
 AC_CHECK_HEADERS([curses.h ncurses/curses.h])
 AC_CHECK_LIB([ncurses], [initscr], [have_ncurses="yes"])
 AM_CONDITIONAL([HAVE_NCURSES], [test "x$have_ncurses" = xyes])
+# Defines libncurses_LIBS that we can use to link against ncurses
+# Needed since >=ncurses-6 where -ltinfo is needed in addition to -lncurses
+PKG_CHECK_MODULES(libncurses, [ncurses >= 5])
+AM_CONDITIONAL([HAVE_NCURSES_PKGCONFIG], [test "$libncurses_LIBS" != ""])
```

Comment:
I think this would fail for some theoretical system which had ncurses support built in, so didn't require a flag, so you'd fall back to adding one which would upset it. See avahi for a better method, where we set a status if PKG_CHECK_MODULES finds the module, and then test that to set the conditional.

You shouldn't need to change the Makefile code, just configure to fix that.

Reply:
Mh, not sure if I can follow. The current solution (before this PR), checks for libncurses and adds `-lncurses` here: https://github.com/OpenLightingProject/ola/pull/1621/files#diff-83cb084b35952ddc152db3cfbc85b9f9L94.
So wouldn't that fail in the current `master` as well?